src/components/sections/About.tsx

```diff
@@ -1,11 +1,44 @@
 import React from 'react';
-import { H1, Section } from '../common/Typography';
+import { H1 } from '../common/Typography';
+import { AboutUs, EventStats } from '@/data/About';
 
 const About = () => {
   return (
-    <Section id="about">
-      <H1>About</H1>
-    </Section>
+    <section
+      id="about"
+      className="relative flex h-screen w-full flex-col items-center justify-center"
+    >
+      <H1 className="leading-normal tracking-[0.12vw] lg:absolute lg:left-[10vw] lg:top-[8.5vw] xl:top-[10vw]">
+        About
+      </H1>
+      <p className="w-[80vw] pb-3 text-lg text-xl font-light leading-normal lg:absolute lg:left-[10vw] lg:top-[14vw] lg:w-[35vw]">
+        {AboutUs}
+      </p>
+
```

Comment:
if you could reuse the Section component that would be great, the section import adds `'h-screen w-full px-8 py-20 md:p-20` by default you can still overwrite the class name using className so you can still add `flex flex-col` etc after that.

e.g

```ts
 <Section
      id="about"
      className="relative flex flex-col items-center justify-center"
    >
</Section>
```